Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -178,8 +198,12 @@ public struct Conv2D<Scalar: TensorFlowFloatingPoint>: Layer {
     /// - Returns: The output.
     @differentiable
     public func callAsFunction(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
-        return activation(conv2D(input, filter: filter, strides: (1, strides.0, strides.1, 1),
-                                 padding: padding) + bias)
+        return activation(conv2D(
+                            input,
```

Comment:
The indentation for input should just be 4-spaces further than the `return` statement.

Reply:
Sorry about that. I was unsure what to do and should've asked beforehand.